apps/voting/contracts/Voting.sol

```diff
@@ -79,8 +84,8 @@ contract Voting is IForwarder, AragonApp {
     * @param _token MiniMeToken Address that will be used as governance token
     * @param _supportRequiredPct Percentage of yeas in casted votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
     * @param _minAcceptQuorumPct Percentage of yeas in total possible votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
-    * @param _voteTime The duration of the main vote phase, i.e. seconds that a vote will be open for token holders to both support and object to the outcome
-    * @param _objectionTime The duration of the objection vote phase, i.e. seconds that a vote will be open after the main vote phase ends for token holders to object to the outcome
+    * @param _voteTime The duration of voting in seconds.
+    * @param _objectionTime The duration of the objection vote phase, i.e. seconds that a vote will be open after the main vote phase ends for token holders to object to the outcome. Main phase duration is calculated as `voteTime - objectionTime`.
     */
     function initialize(MiniMeToken _token, uint64 _supportRequiredPct, uint64 _minAcceptQuorumPct, uint64 _voteTime, uint64 _objectionTime)
```

Comment:
I prefer to have `_yeaAndNayVoteTime` and `_objectionTime`, it would be easear to read. At least in the initialize function params

Reply:
@ujenjt `_voteTime` is now the duration of the whole vote, including the objection phase